### PR TITLE
Improve reusability

### PR DESCRIPTION
**Pull request by dafyddj**
### PR progress checklist (to be filled in by reviewers)

- [x] Changes to documentation are appropriate (or tick if not required)
- [x] Changes to tests are appropriate (or tick if not required)
- [x] Reviews completed

---

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [ ] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [ ] `[ci]`       Changes to the continuous integration configuration
- [ ] `[feat]`     A new feature
- [x] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [ ] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [x] `[docs]`     Documentation changes
- [ ] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Describe the changes you're proposing

Further improvements based on comments in [#180](https://github.com/saltstack-formulas/template-formula/pull/180#issuecomment-565334227), including an usage guide and removal of elements not needed in a derivative formula.

### Documentation checklist

- [x] Updated the `README` (e.g. `Available states`).
- [ ] Updated `pillar.example`.

### Testing checklist

- [ ] Included in Kitchen (i.e. under `state_top`).
- [x] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [ ] Updated the relevant test pillar.

### CODE REVIEW

File: docs/README.rst [hunk 1]
```diff
@@ -45,7 +45,33 @@ Please see `How to contribute <https://github.com/saltstack-formulas/.github/blo
...
+In the rest of this example we'll use ``example`` as the ``<formula theme>``.
+
+Follow these steps to complete the conversion from ``template-formula`` to ``example-formula``.::
+
+  $ git clone git@github.com:YOUR-USERNAME/example-formula.git
+  $ cd example-formula/
+  $ bin/convert-formula.sh example
+  $ git push --force
+
+Alternatively, it's possible to clone ``template-formula`` into a new repository and perform the conversion there. For example:::
```
Comment (myii) [inline docs/README.rst]: One too many colons here, as seen on the [rendered page](https://github.com/saltstack-formulas/template-formula/blob/787b3bc9ed55a2f4e9b202695ed5cc9e505e985b/docs/README.rst#using-this-template).

Comment (dafyddj) [reply docs/README.rst]: Ah yes, I'd misread the reStructuredText docs.

**Comment by myii** [comment]
Specific comment that's been resolved by this PR:

* https://github.com/saltstack-formulas/template-formula/pull/180#issuecomment-565334227

**Comment by myii** [comment]
Merged, brilliant work @dafyddj!

**Comment by daks** [comment]
nice :)

**Pull request merged**